src/services/routes/org.routes.ts

```diff
@@ -25,27 +37,24 @@ export default function registerRoutes(
     },
 
     async handler(request, reply): Promise<void> {
-      const { name, awalaEndpoint } = request.body;
-
-      const isNameValid = isValidDomain(name, { allowUnicode: true });
-      if (!isNameValid) {
-        await reply.code(HTTP_STATUS_CODES.BAD_REQUEST).send({
-          type: ProblemType.MALFORMED_ORG_NAME,
-        });
-        return;
-      }
-
-      if (awalaEndpoint !== undefined && !isValidDomain(awalaEndpoint, { allowUnicode: true })) {
-        await reply.code(HTTP_STATUS_CODES.BAD_REQUEST).send({
-          type: ProblemType.MALFORMED_AWALA_ENDPOINT,
-        });
+      const result = await createOrg(request.body, {
+        logger: this.log,
+        dbConnection: this.mongoose,
+      });
+      if (result.didSucceed) {
+        await reply
+          .code(HTTP_STATUS_CODES.OK)
+          .header('Content-Type', 'application/json')
+          .send(formUrls(result.result.name));
         return;
       }
 
       await reply
-        .code(HTTP_STATUS_CODES.OK)
+        .code(ORG_ROUTES_ERROR_MAPPING[result.reason])
         .header('Content-Type', 'application/json')
```

Comment:
Same here.

Reply:
OK is very specific for the success case. we will not have reason in this case

Reply:
Note I'm referring to the `Content-Type`... I think that if we drop that line, Fastify will use `application/json; charset=utf-8`, which is even better.